variables.tf

```diff
@@ -15,6 +15,14 @@ variable "auth_client_secret" {
   description = "Client Secret for GitHub Auth"
 }
 
+variable "log_level" {
+  description = "The level of log severity (e.g. DEBUG, INFO, ERROR etc)"
+}
+
+variable "loggly_token" {
+  description = "Token for Loggly"
```

Comment:
Less part of this exercise, but this (and several other variables here) should be marked sensitive like your webhook output variable in order to prevent them from being logged in terraform's output
[See the docs](https://learn.hashicorp.com/tutorials/terraform/sensitive-variables#refactor-database-credentials) if you're interested